app/controllers/api/v2/base_controller.rb

```diff
@@ -18,7 +18,7 @@ class BaseController < Api::BaseController
 
       def_param_group :search_and_pagination do
         param :search, String, :desc => N_("filter results")
-        param :order, String, :desc => N_("Sort field and order, eg. ‘id DESC’")
+        param :order, String, :desc => N_("Sort and order by a searchable field, e.g. 'login DESC'")
```

Comment:
I think `name` or `id` might be a better generic example, `login` only exists for users and the same help message is displayed everywhere